.github/workflows/ci.yml

```diff
@@ -280,3 +280,27 @@ jobs:
         with:
           sarif_file: results.sarif
           category: zizmor
+
+  result:
```

Comment:
nit: maybe something like `all-checks` or `required-checks` would be easier to understand

Reply:
Sure, done.

Reply:
nit: update commit message to match